core/src/components/picker-column/picker-column.tsx

```diff
@@ -481,6 +511,159 @@ export class PickerColumn implements ComponentInterface {
     });
   }
 
+  /**
+   * Find the next enabled option after the active option.
+   * @param stride - How many options to "jump" over in order to select the next option.
+   * This can be used to implement PageUp/PageDown behaviors where pressing these keys
+   * scrolls the picker by more than 1 option. For example, a stride of 5 means select
+   * the enabled option 5 options after the active one. Note that the actual option selected
+   * may be past the stride if the option at the stride is disabled.
+   */
+  private findNextOption = (stride: number = 1) => {
```

Comment:
~ `: number` type is redundant when assigning a primitive value to a property.

Reply:
fixed in https://github.com/ionic-team/ionic-framework/pull/29371/commits/294d7712ce73d1a03cfa33d81ba604d7b2385066